examples/GetAncestors/Program.cs

```diff
@@ -0,0 +1,56 @@
+using CommandLine;
+using Concordium.Sdk.Client;
+using Concordium.Sdk.Types;
+
+// We disable these warnings since CommandLine needs to set properties in options
+// but we don't want to give default values.
+#pragma warning disable CS8618
+
+namespace GetAncestors;
+
+internal sealed class GetAncestorsOptions
+{
+    [Option(HelpText = "URL representing the endpoint where the gRPC V2 API is served.",
+        Default = "http://node.testnet.concordium.com:20000/")]
+    public string Endpoint { get; set; }
+    [Option(
+        'm',
+        "max-ancestors",
+        HelpText = "The maximum number of ancestors returned.",
+        Required = true
+    )]
+    public ulong MaxAncestors { get; set; }
+    [Option(
+        'b',
+        "block-hash",
+        HelpText = "Block hash of the block."
+    )]
+    public string BlockHash { get; set; }
+}
+
+public static class Program
+{
+    /// <summary>
+    /// Example how to use <see cref="ConcordiumClient.GetAncestors"/>
+    /// </summary>
+    public static async Task Main(string[] args) =>
+        await Parser.Default
+            .ParseArguments<GetAncestorsOptions>(args)
+            .WithParsedAsync(Run);
+
+    private static async Task Run(GetAncestorsOptions o)
+    {
+        using var cts = new CancellationTokenSource(TimeSpan.FromSeconds(10));

```

Comment:
This is already bound by `MaxAncestors`, so I don't think the cancellation token is interesting, in this example.